libs/scripts/scripts.js

```diff
@@ -48,6 +78,7 @@ export function loadDelayed() {
 async function loadPage() {
   const blocks = decorateArea();
   const navs = decorateNavs();
+  await fetchLib('icon');
```

Comment:
This should 100% not be before LCP. At the very least, not until we do major research on the performance impact.

Icons will not be used all the time and they shouldn't be considered more important than LCP.